packages/core/test/integration/query-interface/bulk-delete.test.ts

```diff
@@ -0,0 +1,105 @@
+import { expect } from 'chai';
+import { DataTypes } from '@sequelize/core';
+import { sequelize } from '../support';
+
+const queryInterface = sequelize.queryInterface;
+
+describe('QueryInterface#bulkDelete', () => {
```

Comment:
Are there currently no `bulkDelete` integration tests?

Reply:
I couldn't find any hence the reason for adding this in